.github/workflows/molecule-tests.yml

```diff
@@ -0,0 +1,124 @@
+---
+name: Molecule tests
+
+on:
+  push:
+  pull_request:
+
+env:
+  CARTRIDGE_CLI_VERSION: '2.4.0'
+
+jobs:
+  molecule-tests-ce:
+    if: |
+      github.event_name == 'push' ||
+      github.event_name == 'pull_request' && github.event.pull_request.head.repo.owner.login != 'tarantool'
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        tarantool-version: ["1.10", "2.5"]
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@master
+        with:
+          path: '${{ github.repository }}'
+
+      - name: Setup python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+
+      - name: Install molecule requirements
+        run: |
+          pip3 install --upgrade \
+            ansible==2.8 \
+            molecule==3.0.2 \
+            docker \
+            flake8==3.8.1 \
+            testinfra
+
+      - name: Cache test packages
+        id: cache-packages
+        uses: actions/cache@v2
+        with:
+          path: '${{ github.repository }}/packages'
+          key: ce-${{ matrix.tarantool-version }}-${{ env.CARTRIDGE_CLI_VERSION }}
+
+      - name: Create test packages
+        if: steps.cache-packages.outputs.cache-hit != 'true'
+        working-directory: '${{ github.repository }}'
+        run: |
+          sudo apt -y update
+          sudo apt -y install git gcc make cmake unzip
+          git config --global user.email "test@tarantool.io" \
+            && git config --global user.name "Tar Antool"
+
+          curl -L https://tarantool.io/JRceXH/release/${{ matrix.tarantool-version }}/installer.sh | bash
```

Comment:
Should work without `JRceXH`

Reply:
This command is copied from [tarantool.io download](https://www.tarantool.io/en/download/os-installation/ubuntu/) instructions. I believe that it should be stable.

Reply:
Removed `JRceXH `.